examples/generic_connector_configs/elasticsearch_secretless.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+services:
+  token-service:
+    connector: generic_http
+    listenOn: tcp://0.0.0.0:9030
+    credentials:
+      token:
+        from: keychain
+        get: summon#elasticsearch/access_token
+    config:
+      headers:
+        Authorization: Bearer {{ .token }}
```

Comment:
Shouldn't the token value also be base64 encoded?

Reply:
According to their documentation, it shouldn't be

Reply:
Ah OK. Based on [this documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-get-token.html) the Bearer token must be base64 encoded. I guess in this case the user will have stored the token base64 encoded already.

I think it would be worth stating this clearly in a comment to avoid confusion e.g. having no bas64 encoding or double encoding.

Reply:
According to that documentation, it seems that only kerberos tickets as a parameter involve base 64. It doesn't mention using base 64 for bearer tokens as far as I can tell.